frontend/src/components/doctor/DoctorControl.tsx

```diff
@@ -10,12 +15,17 @@ export type Doctor = {
   email: string;
   gender: string;
   location: string;
-  speciality: string;
+  specialty: string;
+  status: boolean;
+  about: string;
```

Comment:
If you can, I'd try to reuse the `Doctor` type exported from `backend/src/model/queries.ts`.